.gitignore

```diff
@@ -51,3 +51,7 @@ jspm_packages/
 .yarn/build-state.yml
 .yarn/install-state.gz
 .pnp.*
+
+# IDE directories
+.vscode/
+.idea/
```

Comment:
Not necessary, please remove this change. You can add your own exclusions in a global `.gitignore` on your host.

Reply:
Generally, it can be covered by options in global `.gitignore`, but some projects can store IDE's specific folders inside the repository. But in this case, I will remove it.